packages/ui/src/radio-group/RadioGroup.test.tsx

```diff
@@ -30,12 +30,7 @@ const RADIO_GROUP_OPTIONS = [
   { label: "1 storey", value: OPTION_1_VALUE },
   { label: "2 storeys", value: OPTION_2_VALUE },
   {
-    label: (
-      <>
-        3&nbsp;
-        <Button variant={"glossary"}>storeys</Button>
-      </>
-    ),
+    label: '3 <Button variant="glossary">storeys</Button>',
```

Comment:
typo?

Reply:
No sadly, that's the Canadian way they spell story in the Building Code so I decided to keep the example as such.